boards/arm/nrf54l15dk_nrf54l15/Kconfig.defconfig

```diff
@@ -0,0 +1,22 @@
+# Copyright (c) 2024 Nordic Semiconductor ASA
+# SPDX-License-Identifier: Apache-2.0
+
+if BOARD_NRF54L15DK_NRF54L15_CPUAPP
+
+config BOARD
+	default "nrf54l15dk_nrf54l15_cpuapp"
+
+config ENTROPY_GENERATOR
+	default y if CRACEN
+
+endif # BOARD_NRF54L15DK_NRF54L15_CPUAPP
+
+config BT_CTLR
+	default BT
+
+# MPU-based null-pointer dereferencing detection cannot
+# be applied as the (0x0 - 0x400) is unmapped for this target.
```

Comment:
if this cannot be applied, then probably better to explicitly set it in `_defconfig` instead.

Reply:
I cannot move it to the `_defconfig` file, it causes a compilation error (undefined symbol).

Reply:
> I cannot move it to the _defconfig file, it causes a compilation error (undefined symbol). Even if I leave the definition of NULL_POINTER_EXCEPTION_DETECTION here.

what can you not move ?
is it `NULL_POINTER_EXCEPTION_DETECTION_NONE` ?

I can do following in `_defconfig` without seeing any errors:
```
CONFIG_NULL_POINTER_EXCEPTION_DETECTION_NONE=y
```

Reply:
Oh, then my bad, I was trying to set `NULL_POINTER_EXCEPTION_DETECTION` with `NULL_POINTER_EXCEPTION_DETECTION_NONE`. In that case, I will move that config to `_defconfig` file.